frontend/src/index.css

```diff
@@ -5,9 +5,121 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #fddfe7;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.charCount {
+  color: #0425ff;
+}
+
+.charcountWrapper {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 60px;
+  margin-right: 230px;
+}
+
+h1 {
+  font-size: 36px;
+  color: #0425ff;
+}
+
+h3 {
+  font-size: 24px;
+  margin-bottom: 0;
+  color: #0425ff;
+}
+
+h4 {
+  font-size: 24px;
+  margin-bottom: 0;
+  width: 50%;
+  margin-top: 50px;
+}
+
+section {
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+button {
+  font-size: 20px;
+  appearance: none;
+  border-radius: 40px;
+  background: #f93526;
+  color:#fddfe7;
+  border: none;
+  padding: 10px 30px;
+  font-weight: bold;
+  margin-top: 30px;
+  margin-left: 10px;
+  cursor: pointer;
+}
```

Comment:
I noticed that when I entered an invalid email address and tried to enter my password, the red border that appeared was almost unnoticeable due to the red border that was already around the button.  How could we make this clearer to the user?